app/process/bikelanes/categories.lua

```diff
@@ -139,6 +139,22 @@ local function cyclewayOnHighway(tags)
   end
 end
 
+local function cyclewayBetweenLanes(tags)
+  -- TODO: maybe we want this only on transformed objects e.g. _parent_highway ~= nil
+  -- TODO: add description
+  local result = false
+  if tags['cycleway:lanes'] and string.find(tags['cycleway:lanes'], "|lane|", 1, true) then
+    result = true
+  end
+  if tags['bicycle:lanes'] and string.find(tags['bicycle:lanes'], "|designated|", 1, true) then
+    result = true
+  end
+  if result then
+    return "cyclewayBetweenLanes"
+  end
```

Comment:
Ich denke, in diesem Fall können wir es kurz halten…

```suggestion
  if tags['cycleway:lanes'] and string.find(tags['cycleway:lanes'], "|lane|", 1, true)
    or tags['bicycle:lanes'] and string.find(tags['bicycle:lanes'], "|designated|", 1, true) then
    return "cyclewayBetweenLanes"
  end
```

Sieht sonst gut aus, gerne dann nach Pollieren direkt mergen (und Ticket dann an mich für die Fronten-Themen)

Reply:
Yes we can!